Sources/GDSCommon/Patterns/README.md

```diff
@@ -253,7 +252,7 @@ If the viewModel conforms to BaseViewModel:
 
 ```swift
 struct MockErrorViewModel: GDSErrorViewModel, BaseViewModel {
```

Comment:
These docs aren't quite right anymore 🙂

Reply:
Updated, lmk if you think this needs more/clearer detail 🙂